draft-ietf-mls-protocol.md

```diff
@@ -1614,14 +1615,21 @@ struct {
 } MLSPlaintextTBS;
 ~~~~~
 
-<!-- OPEN ISSUE: group_id and epoch are duplicated in the TBS and in
-GroupContext. Think about how to de-duplicate. -->
```

Comment:
You didn't fix this issue?

Reply:
I thought about it, and decided not to worry about it.  If you feel strongly, we can keep it.